api/src/main/java/org/apache/cloudstack/api/command/admin/config/ListCfgsByCmd.java

```diff
@@ -172,8 +183,11 @@ public void execute() {
             if (getDomainId() != null) {
                 cfgResponse.setScope("domain");
             }
-            if (getImageStoreId() != null){
-                cfgResponse.setScope("imagestore");
+            if (getImageStoreId() != null) {
+              cfgResponse.setScope("imagestore");
+            }
+            if (getNetworkId() != null) {
+                cfgResponse.setScope(ConfigKey.Scope.Network.name());
```

Comment:
@soreana , you have a compile error here:
```
08:58:05 [ERROR] COMPILATION ERROR : 
08:58:05 [INFO] -------------------------------------------------------------
08:58:05 [ERROR] /jenkins/workspace/acs-centos7-pkg-builder/dist/rpmbuild/BUILD/cloudstack-4.18.0.0-SNAPSHOT/api/src/main/java/org/apache/cloudstack/api/command/admin/config/ListCfgsByCmd.java:[190,52] error: package ConfigKey.Scope does not exist
```

Reply:
Thanks 👍
I'm checking that.

Reply:
It is fixed now. I missed an import. That might be due to a recent rebase with the main branch.